example/example.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+require "../vendor/autoload.php";
+
+use Symfony\Component\Yaml\Yaml;
+use yswery\DNS\Resolver\JsonResolver;
+use yswery\DNS\Resolver\RecursiveResolver;
+use yswery\DNS\Resolver\StackableResolver;
+use yswery\DNS\Server;
+
+// instantiate resolver
+$resolver = new StackableResolver(
+    [
+        new JsonResolver('../config/dns.example.json'), // JSON formatted DNS records file
+        new RecursiveResolver(),                         // Recursive provider acting as a fallback to the JsonStorageProvider
+    ]
+);
+
+// load configuration
+$config = Yaml::parseFile('../config/config.yml');
+
```

Comment:
As is, there is no console output when running the example. Looks like we need something like this inserted here...

`
$server = new Server($resolver, $config);
$output = new \Symfony\Component\Console\Output\ConsoleOutput();
$logger = new \yswery\DNS\Event\ConsoleEventSubscriber($output);
$server->registerEventSubscriber($logger);

// Creating a new instance of server and start it
$server->run();
`

Reply:
Please use `bin/phpdns` it will show error, it will also show logs that needs improvement when run with -vv param.